tests/status.py

```diff
@@ -72,6 +72,62 @@ def runTest(self):
         board.expect(prompt)
         self.result_message = 'ifconfig shows ip addresses: %s' % tmp
 
+class MacAddressCheck(rootfs_boot.RootFSBootTest):
+    '''Checked MAC addresses for all interfaces.'''
+    def __checkMac(self, num1, num2, count):
```

Comment:
num1, num2 are not descriptive enough. mac_address, base_mac instead? Count is slightly more descriptive, but it'd make sense to rename it to something that makes it more obvious